### PR TITLE
Add lnms maintenance cleanup command and fix alert rule UI (delete) generating orphans

### PR DESCRIPTION
**Pull request by Calvario**
Please give a short description what your pull request is for

Delete alerts & alert_log when rule is deleted on the UI.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
@Calvario perhaps a one time cleanup option?

**Comment by Calvario** [comment]
@murrant like a database seeder?

**Comment by murrant** [comment]
Maybe a new `lnms maintenance:` command (intended to hold other database cleanups as well)

**Comment by Calvario** [comment]
@murrant ready for review :)

**Comment by Calvario** [comment]
@murrant is it as expected ?

File: includes/html/forms/delete-alert-rule.inc.php [hunk 1]
```diff
@@ -31,6 +31,10 @@
         $alert_msg_prefix .= ' id ' . $vars['alert_id'];
     }
     if (dbDelete('alert_rules', '`id` =  ?', [$vars['alert_id']])) {
+        // @phpstan-ignore-next-line
```
Comment (murrant) [inline includes/html/forms/delete-alert-rule.inc.php:34]: db*() functions are deprecated and will be deleted.  Let's try not to add new lines using them. :)

Alert::where('rule_id', $vars['alert_id'])->delete();
AlertLog::where('rule_id', $vars['alert_id'])->delete();

Comment (Calvario) [reply includes/html/forms/delete-alert-rule.inc.php:34]: Agree, as AlertTemplateMap has a class too, I changed it too.

**Comment by laf** [comment]
My only comment on this is the new command is generically named but has a specific job around alerting. If we plan to add more cleanup queries within the same command then that will be fine, thoughts @murrant?

**Comment by Calvario** [comment]
Hi @murrant, is something missing on this PR?

**Review by laf — APPROVED** [review]
LGTM.

**Pull request merged**